vars/publishPerformanceReports.groovy

```diff
@@ -1,13 +1,13 @@
 import uk.gov.hmcts.contino.Gatling
 
-def call(steps, params)  {
+def call(params)  {
   try {
-      azureBlobUpload('buildlog-storage-account', steps.env.GATLING_REPORTS_DIR, "performance/${params.product}-${params.component}/${params.environment}")
+      azureBlobUpload('buildlog-storage-account', env.GATLING_REPORTS_PATH, "performance/${params.product}-${params.component}/${params.environment}")
```

Comment:
GATLING_REPORTS_DIR included $WORKSPACE which wasn't getting resolved after switching to single quotes